Ivan_Belov/10.Kubernetes.WS/config

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
Please remove config, even commit with this config

Reply:
Done